src/core/server/ui_settings/ui_settings_client.test.ts

```diff
@@ -239,25 +243,10 @@ describe('ui settings', () => {
   });
 
   describe('#getDefaults()', () => {
-    it('returns a promise for the defaults', async () => {
-      const { uiSettings } = setup();
-      const promise = uiSettings.getDefaults();
-      expect(promise).to.be.a(Promise);
-      expect(await promise).to.eql({});
-    });
-  });
-
-  describe('getDefaults() argument', () => {
-    it('casts sync `getDefaults()` to promise', () => {
```

Comment:
Not sure why it was done in this manner. Probably for the sake of consistency with other accessing methods.